src/components/play/PlayUI.js

```diff
@@ -1874,18 +1877,18 @@ class PlayUI extends Component {
     setIsPlayingSequenceGraphic = ({ x, y }) => {
         const { user, round } = this.props
         const isPlayingSequence = round.userPatterns[this.props.user.id].isPlayingSequence
-        const switchLetterContainer = this.container.nested().circle(28)
-        let switchLetterContainerX = x + 195
-        let switchLetterContainerY = y + 150
+        const switchLabelContainer = this.container.nested().circle(28)
```

Comment:
what's 28?

Reply:
Its the switchLabelContainer size, moved it to constants

Reply:
> What is this doing?
When a user is recording a sequence, they have 8 slots available, the app is supposed to automatically stop the sequence recording once all the 8 are full, it wasn't doing it. This fixes that bug.